src/attribution/onINP.ts

```diff
@@ -167,7 +181,7 @@ const cleanupEntries = () => {
   // events are dispatched out of order. When this happens they're generally
   // only off by a frame or two, so keeping the most recent 50 should be
   // more than sufficient.
```

Comment:
This is the comment I'm suggest now belongs with the MAX_PENDING_ENTRIES definition.